apps/extension/webpack/webpack.common.js

```diff
@@ -70,13 +74,40 @@ export default {
     },
   },
   plugins: [
+    new webpack.CleanPlugin(),
     new webpack.ProvidePlugin({
       // Required by the `bip39` library
       Buffer: ['buffer', 'Buffer'],
     }),
+    new webpack.IgnorePlugin({
+      // Not required by the `bip39` library, but very nice
+      checkResource(resource) {
+        return /.*\/wordlists\/(?!english).*\.json/.test(resource);
+      },
+    }),
```

Comment:
Does this ignore any bip39 wordlist that isn't english? Won't this make our validator non-comprehensive and prevent users from adding valid phrases in other languages?

Reply:
yes, this ignores any bip39 wordlist that isn't in english.

previously, every bip39 wordlist in every language was bundled into every script in both webapp and extension. they are not insignificant so it was a good target to reduce js bundle size.

we don't currently generate any phrases that aren't english, so we won't ever give an extension user a phrase we can't validate.

we shouldn't enforce validation on phrases that the user inputs, as instructed by the library, and as issued in #425

the bip39 library we are using specifically and prominently recommends this approach and this webpack configuration, at the top of their readme, which is linked in that issue

if at some point we support additional languages, we should chunk languages separately, and not load bip39 unless we are performing phrase validation

Reply:
it looks like we eventually use the seed phrase in wasm, which uses SeedPhrase::from_str, which uses SeedPhrase::validate_checksum, which checks for presence in a list which is exactly equivalent to our english list from the bip39 package.

https://github.com/penumbra-zone/penumbra/blob/66b13e98643b8af8285238c62657ded81e8824f1/crates/core/keys/src/keys/seed_phrase/words.rs 

i think enforcing validation on a bip39 phrase input is wrong, but if we are going to validate, we had better only validate english words.

the current state of this pr satisfies that, so i'm keeping it as-is

Reply:
Oh! If our systems currently only support english, yep, we should just validate for that then. I suppose it's a bigger task for later to support more languages then.